.changeset/forty-timers-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
wasn't sure if this is a minor change or just a patch change. Lemme know and I'll change it accordingly! I do suspect that this shouldn't break anything.